.github/workflows/build.yml

```diff
@@ -66,7 +66,7 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: install system dependencies
-      run: sudo apt-get update && sudo apt-get install libelf-dev mpich libmpich-dev libomp-11-dev libsystemd-dev python3-gi python3-yaml
+      run: sudo apt-get update && sudo apt-get install libelf-dev mpich libmpich-dev libomp-11-dev libsystemd-dev liburing-dev python3-gi python3-yaml
```

Comment:
please add to

https://github.com/geopm/geopm/blob/dev/service/debian/control#L5-L18

as well!